Contacts/People/people.html

```diff
@@ -24,40 +24,6 @@ <h3>
   present distribution.
 </p>
 
-<h3>
-  Module authors and maintainers
-</h3>
-<p>
-  It is also clear that the contributions of different people vary
-  significantly in size. Therefore and in view of the total size of the
-  GAP library we decided in 2001 to introduce the concept
-  of 'modules'. A module is a part of GAP that
-</p>
-<ul>
-  <li> provides a describable functionality, </li>
-  <li> consists of a discrete set of files, </li>
-  <li> has a well-defined set of authors and maintainers. </li>
-</ul>
-<p>
-  On a separate page we provide a list of these
-  <a href="modules.html">Modules</a> in which for each module we give
-  the authors and maintainers as well as a brief description of its
-  functionality in one line. 
-</p>
-<p>
-  There are some GAP authors with substantial contributions
-  to GAP that are not definable as modules, but may be even
-  considered as the backbone of the system. Of these, in the first place
-  Martin Schönert should be mentioned who has been the system architect
-  of GAP from its beginning until 1996. In that period
-  (until 1997) also Frank Celler has made large contributions to the system.
-  Also from the beginning of GAP and until now, Thomas
-  Breuer has made large contributions not only to his special area of
-  work, namely representation theory in GAP. Steve Linton,
-  Alexander Hulpke, and Frank Lübeck have taken over a good deal of
-  Martin's rôle since 1997.
```

Comment:
So the one thing to mention is that this kind of thing could have been useful for grant applications. It still might be, if it was updated to reflect the current reality... But perhaps there are other better ways to do that these days, e.g. using the GitHub "insights" features?